phpBB/language/en/install.php

```diff
@@ -439,6 +440,7 @@
 	'FILES_NOT_MODIFIED_EXPLAIN'	=> 'The following files are not modified and represent the original phpBB files from the version you want to update from.',
 	'FILES_UP_TO_DATE'				=> 'Already updated files',
 	'FILES_UP_TO_DATE_EXPLAIN'		=> 'The following files are already up to date and do not need to be updated.',
+	'FILES_VERSION'					=> 'Files version',
```

Comment:
Files version doesn't sound right. Either "Package version" or "Package version installed"

I would also use `PACKAGE_VERSION` instead of `FILES_VERSION` here and elsewhere.